v4/integration/dydxStateManager/src/main/java/exchange/dydx/dydxstatemanager/MarketExtensions.kt

```diff
@@ -0,0 +1,10 @@
+package exchange.dydx.dydxstatemanager
+
+import exchange.dydx.abacus.output.PerpetualMarket
+
+val PerpetualMarket?.maxLeverage: Double
+    get() {
+        if (this == null) return 5.0
+        val imf = configs?.run { effectiveInitialMarginFraction ?: initialMarginFraction }
+        return imf?.let { 1.0 / it } ?: 5.0
```

Comment:
1x might be the safer default value here in case the market's open interest is off the charts.

additionally, might want a div by 0 safety check